static/app/utils/theme.tsx

```diff
@@ -865,7 +865,16 @@ type MyTheme = Theme;
  */
 declare module '@emotion/react' {
   // eslint-disable-next-line @typescript-eslint/no-shadow
-  export interface Theme extends MyTheme {}
+  export interface Theme extends MyTheme {
+    /**
+     * [For Storybook only, do not use inside the app.]
+     *
+     * Substitute for the "background" property in Storybook components
+     * (anything inside docs-ui), since Storybook overrides "background" with
+     * its own object value.
+     */
+    bg?: string;
+  }
```

Comment:
Extending Theme here so we don't get typescript warnings when using `theme.bg` in `docs-ui`.

Reply:
I would declare this somewhere in our storybook files, otherwise it will be confusing for people working on the sentry app.